python/fastquant/data/stocks/pse.py

```diff
@@ -321,6 +321,9 @@ def get_pse_data(
     if save:
         pse_data_df.to_csv(fp, index=False)
         print("Saved: ", fp)
+    # add dividend column for dividends sake XD
+
+    pse_data_df["dividend"] = 0
```

Comment:
@rafmacalaba Ah yeah, so I get now why there's an error when there's no dividend column. It's because the column adding happens at the "get data" level. This should be done on the `backtest` level instead, so it's applied to dataframe inputs regardless of source.